tests/lsdb/catalog/test_catalog.py

```diff
@@ -109,6 +109,22 @@ def test_query_no_arguments(small_sky_order1_catalog):
         small_sky_order1_catalog.query(None)
 
 
+def test_query_margin(small_sky_xmatch_with_margin):
+    expected_ddf = small_sky_xmatch_with_margin._ddf.copy()
+    expected_ddf = expected_ddf[
```

Comment:
Could you collapse this into a single assign like you did for `expected_margin_ddf`?